packages/reporter/src/commands/commands.scss

```diff
@@ -45,14 +45,24 @@
   }
 
   // Child Styles
-  .command-type-child {
-    .command-method {
-      &:before {
-        float: left;
-        content: "-";
-        margin-right: 2px;
-        padding-left: 5px;
-      }
+  .command-type-parent > span > .command-wrapper {
```

Comment:
@lmiller1990 These styles can be removed. Theres be a pretty large refactor the command log composition in 10.0-release branch so these don't align. If you look at percy, these are removing the child command styles.

I merged with 10.0 from dev earlier..hopefully this isn't a continue issue.

Reply:
@emilyrohrbough I think the last merge was a squash merged rather than as a merge commit which is why this needed to be revisited

Reply:
@tgriesser ah shoot.. you're right.